### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by QUERY-GANI**
* add try catch in plugins/online.js
* add usedPrefix in plugins/broadcastbot.js
* add leave group feature
* edit plugins/demote.js

### CODE REVIEW

**Comment by FadliDarmawan** [comment]
Wow

**Comment by KazukoGans** [comment]
> * add try catch in plugins/online.js
> * add usedPrefix in plugins/broadcastbot.js
> * add leave group feature
> * edit plugins/demote.js

Tysm fitur nya sangat membantu bang

File: plugins/promote.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-let handler = async (m, { conn, args }) => {
+let handler = async (m, { conn }) => {
   let users = m.mentionedJid
-  conn.groupMakeAdmin(m.chat, users)
+  if (!users.length) m.reply("tag membernya!")
+  else conn.groupMakeAdmin(m.chat, users)
```
Comment (Nurutomo) [inline plugins/promote.js:4]: Rawan banned kalo nggk satu per satu

File: plugins/opromote.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-let handler = async (m, { conn, args }) => {
+let handler = async (m, { conn }) => {
   let users = m.mentionedJid
-  conn.groupMakeAdmin(m.chat, users)
+  if (!users.length) m.reply("tag membernya!")
+  else conn.groupMakeAdmin(m.chat, users)
```
Comment (Nurutomo) [inline plugins/opromote.js:4]: Rawan banned kalo nggk satu per satu

File: plugins/leave.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+let handler = async (m, { conn, args, command }) => {
+    if (/all/g.test(command)) {
+        for (const jid of conn.chats.all().filter(v => v.jid.endsWith('g.us') && !v.archive).map(v => v.jid)) {
+            const sayonara = (await conn.groupMetadata(jid)).id
+            await conn.groupLeave(sayonara)
```
Comment (Nurutomo) [inline plugins/leave.js:5]: Belum tau groupLeave itu kek gimana, pas aku cek seperti mengkick diri sendiri...tpi nanti ku coba lagi

File: plugins/demote.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-let handler = async (m, { conn, args, usedPrefix }) => {
+let handler = async (m, { conn }) => {
   let users = m.mentionedJid
-  for (let user of users) conn.groupDemoteAdmin(m.chat, user).catch(console.log)
+  if (!users.length) m.reply("tag membernya!")
+  else conn.groupDemoteAdmin(m.chat, users)
```
Comment (Nurutomo) [inline plugins/demote.js:4]: Rawan banned kalo nggk satu per satu

File: plugins/broadcastjadibot.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+let handler = async (m, { conn, text, usedPrefix }) => {
   if (conn.user.jid === global.conn.user.jid) {
     let users = [...new Set([...global.conns.filter(conn => conn.user && conn.state !== 'close').map(conn => conn.user.jid)])]
     let cc = text ? m : m.quoted ? await m.getQuotedObj() : false || m
     let teks = text ? text : cc.text
     let content = conn.cMod(m.chat, cc, /bc|broadcast/i.test(teks) ? teks : teks + '\n' + readMore + '「 All Jadibot Broadcast 」')
     for (let id of users) conn.copyNForward(id, content, true)
     conn.reply(m.chat, `_Berhasil mengirim broadcast ke ${users.length} nomor yang jadi bot_
-${users.map(v => 'wa.me/' + v.replace(/[^0-9]/g,'') + '?text=.menu').join('\n')}`.trim(), m)
+${users.map(v => `wa.me/${v.replace(/[^0-9]/g,'')}?text=${usedPrefix}menu`).join('\n')}`.trim(), m)
```
Comment (Nurutomo) [inline plugins/broadcastjadibot.js:9]: encodeURIComponent(usedPrefix)

**Review by Nurutomo — CHANGES_REQUESTED** [review]
Rawan banned kalo nggk satu per satu

File: plugins/leave.js (hunk 3 above)
Comment (ariffb25) [reply plugins/leave.js:5]: fungsinya emang buat keluar dari grup, sesuai docs nya :v 
image

Comment (Joseph92123) [reply plugins/leave.js:5]: itu apa ? ^^

File: plugins/broadcastjadibot.js (hunk 5 above)
Comment (QUERY-GANI) [reply plugins/broadcastjadibot.js:9]: kok di encode bwang?

File: plugins/leave.js (hunk 3 above)
Comment (QUERY-GANI) [reply plugins/leave.js:5]: gw udah coba leave all 20+ gc ga banned sih mungkin gara gara nomor gw dah lama gw pake bot

File: plugins/demote.js (hunk 4 above)
Comment (QUERY-GANI) [reply plugins/demote.js:4]: kalau satu satu error bwang :

TypeError: participants.map is not a function
    at WAConnection.groupQuery (/home/lelouch/wabot-aq/node_modules/@adiwajshing/baileys/lib/WAConnection/8.Groups.js:151:41)
    at WAConnection.groupDemoteAdmin (/home/lelouch/wabot-aq/node_modules/@adiwajshing/baileys/lib/WAConnection/8.Groups.js:128:61)
    at WAConnection.handler (/home/lelouch/wabot-aq/plugins/demote.js:3:32)
    at WAConnection.handler (/home/lelouch/wabot-aq/handler.js:201:26)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

File: plugins/opromote.js (hunk 2 above)
Comment (QUERY-GANI) [reply plugins/opromote.js:4]: opromote.js cuman tambahin if else doang :v

File: plugins/promote.js (hunk 1 above)
Comment (QUERY-GANI) [reply plugins/promote.js:4]: promote.js cuman tambahin if else doang :v

**Review by QUERY-GANI — COMMENTED** [review]
oke

File: plugins/demote.js (hunk 4 above)
Comment (Nurutomo) [reply plugins/demote.js:4]: [user] kalo satu2

File: plugins/broadcastjadibot.js (hunk 5 above)
Comment (Nurutomo) [reply plugins/broadcastjadibot.js:9]: kalo prefixnya `&` gmn ;v

nanti malah gaada teksnya
contoh
https://wa.me/?text=&menu

Comment (QUERY-GANI) [reply plugins/broadcastjadibot.js:9]: iya juga ya v:

File: plugins/demote.js (hunk 4 above)
Comment (QUERY-GANI) [reply plugins/demote.js:4]: owalah oke bwang

**Review by Nurutomo — APPROVED** [review]
oklah :v
yang viewed aja yg di merge

**Comment by Nurutomo** [comment]
dah lah, buat baru aja, conflict :v

**Pull request closed without merging**